src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAccountManager.java

```diff
@@ -1663,6 +1677,27 @@ private void updateLastSeen(final RegisteredUser user) throws SegueDatabaseExcep
         }
     }
 
+    /**
+     * Logout user from other sessions.
+     * Increment the users' session token field and create a new session.
+     *
+     * @param request
+     *            - request containing session information.
+     * @param response
+     *            - response to update cookie information.
+     * @param rememberMe
+     *            - Boolean to indicate whether or not this cookie expiry duration should be long or short
+     * @throws SegueDatabaseException
+     *             - if an error occurs with the update.
+     */
+    public void logoutElsewhere(final HttpServletRequest request, final HttpServletResponse response,
+                                final boolean rememberMe) throws SegueDatabaseException {
+        RegisteredUser user = this.getCurrentRegisteredUserDO(request);
+        this.database.incrementSessionToken(user);
+        user.setSessionToken(user.getSessionToken() + 1);
```

Comment:
This seems to make a dangerous assumption about what the method on the line above does. Since this isn't a common path, would incrementing the session token and then getting the user from the database again not be safest?